src/pages/passwordUpdated.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement } from 'react';
+import { NextPageContext } from 'next';
+import { parseCookies } from 'nookies';
+import TwoThirdsLayout from '../layout/Layout';
+import { deleteCookieOnServerSide } from '../utils';
+import { USER_COOKIE } from '../constants';
+
+const title = 'Password Updated - Fares Data Build Tool';
+const description = 'Password Updated page of the Fares Data Build Tool';
+
+interface PasswordUpdatedProps {
+    redirectTo: string;
+}
+
+const PasswordUpdated = ({ redirectTo }: PasswordUpdatedProps): ReactElement => (
+    <TwoThirdsLayout title={title} description={description}>
+        <h1 className="govuk-heading-l">Password Updated</h1>
+        <p className="govuk-body-l">Your password has been updated successfully</p>
+        <a
+            href={redirectTo}
+            role="button"
+            draggable="false"
+            className="govuk-button"
+            data-module="govuk-button"
+            id="start-now-button"
```

Comment:
should this be continue button?

Reply:
Yep! I've changed this now on all pages where it was off